s3/s3_transport/unit_tests/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ project(unit_tests LANGUAGES C CXX)
 
 find_package(IRODS 4.3.0 EXACT REQUIRED)
 
-set(IRODS_EXTERNALS_FULLPATH_S3 "/opt/irods-externals/libs3e4197a5e-0")    # without verbosity
+set(IRODS_EXTERNALS_FULLPATH_S3 "/opt/irods-externals/libs3e4674774-0")    # without verbosity
```

Comment:
why do we have to redefine this here?   can't we inherit somehow from the main CMakeLists.txt?

@SwooshyCueb any suggestions?  or is this just how it works?

Reply:
Yeah this should not be explicitly defined at all. We have macros for that.

Reply:
Can someone point me to how this would be done?  Also, I need to be able to select the version for development purposes.

Reply:
I changed it to the following which matches what is done in the non-unit test cmake file.

```
include(IrodsExternals)
IRODS_MACRO_CHECK_DEPENDENCY_SET_FULLPATH_ADD_TO_IRODS_PACKAGE_DEPENDENCIES_LIST(S3 libs3e4674774-0)
string(REPLACE ";" ", " IRODS_PACKAGE_DEPENDENCIES_STRING "${IRODS_PACKAGE_DEPENDENCIES_LIST}")
```

Reply:
This is fine, however it is curious to me that it needs to be redefined in the first place

Reply:
Unless you are packaging the unit tests separately, the `IRODS_PACKAGE_DEPENDENCIES_STRING` is not needed.